pkg/apis/kubeone/v1beta2/defaults.go

```diff
@@ -285,6 +288,20 @@ func SetDefaults_Features(obj *KubeOneCluster) {
 	}
 }
 
+func SetDefaults_TLSCipherSuites(obj *KubeOneCluster) {
+	if obj.TLSCipherSuites.APIServer == nil {
+		obj.TLSCipherSuites.APIServer = kubernetesconfigs.TLSCipherSuites(kubernetesconfigs.APIServerDefaultTLSCipherSuites())
+	}
+
+	if obj.TLSCipherSuites.Etcd == nil {
+		obj.TLSCipherSuites.Etcd = kubernetesconfigs.TLSCipherSuites(tls.CipherSuites())
+	}
+
+	if obj.TLSCipherSuites.Kubelet == nil {
+		obj.TLSCipherSuites.Kubelet = kubernetesconfigs.TLSCipherSuites(tls.CipherSuites())
+	}
```

Comment:
Can we reduce this to ciphers demanded by CIS? See https://github.com/aquasecurity/kube-bench/blob/abfa7d9613f0f5f9e628a3ec87fea3443fe57805/cfg/cis-1.8/node.yaml#L420-L442

Reply:
BTW: this list includes 2 insecure cipher suites: `TLS_RSA_WITH_AES_256_GCM_SHA384`, `TLS_RSA_WITH_AES_128_GCM_SHA256`.